e2e/utils/expect.ts

```diff
@@ -0,0 +1,23 @@
+import { expect } from 'chai';
```

Comment:
Does this code duplicate the one written in retry?

Reply:
I've delete retry.ts